### ISSUE TITLE
Run the View Node Package Source command  error ~

### ISSUE THREAD

**Issue by maquannene** [dkundel/vscode-npm-source#6]
When I Run the View Node Package Source command,
the vscode show a error "running the contributed command:'extension.openPackageSource' failed."

how should i go？

**Comment by maquannene** [comment]
oh ,i solve it , it must run in js file.

but i can`t  open the source file

= 。= ~

**Comment by eram** [comment]
yes - pls add support for ts files - these have the same imports as es6
